web_monitoring/db.py

```diff
@@ -760,3 +785,17 @@ def get_version_by_versionista_id(self, versionista_id):
         # Make result look like the result of `get_version` rather than the
         # result of `list_versions`.
         return {'data': result['data'][0]}
+
+    def validate_credentials(self):
+        """
+        Validate that the DB Client is authorized for the provided host
```

Comment:
Same as above, full sentences. Also, I think the important bit to call out here is that this will raise an error in the case of invalid credentials, so it should be used kind of like `assert`.

You can also uses a `Raises` section to call out the specific error type if you want. (See the Numpy docstring styleguide: https://numpydoc.readthedocs.io/en/latest/format.html#sections)

Reply:
The Numpy reference was super helpful, thanks for the link.